### PR TITLE
Added enabled state for the sound items

### PR DESCRIPTION
**Pull request by ocontrerastx**
Known Bug

1. Click on the sound item to enable the sound
2. Click and drag on the slider

The action of clicking on the slider causes the handleSoundItemsClick to be triggered and sets the item to be disabled. 

I need to figure out how I can let a user click in the area (including the slider area) when it is disabled. But then also ignore slider click drag events.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -48,18 +50,72 @@ header {
...
   width: 84px;
+  cursor:pointer;
+  opacity: 0.6;
   display: flex;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
+}
+
+.sounds__item-active {
```
Comment (tabuckner) [inline css/styles.css:63]: I wonder if this more closely aligns with BEM standards:
```suggestion
.sounds__item--active {
```

File: css/styles.css [hunk 2]
```diff
@@ -48,18 +50,72 @@ header {
...
+  border-radius: 5px;  
+  background: #d3d3d3;
+  outline: none;
+  opacity: 1;
+  -webkit-transition: .2s;
+  transition: opacity .2s;
+}
+
+
+.slider::-webkit-slider-thumb {
```
Comment (tabuckner) [inline css/styles.css:89]: on the next project we should introduce either PostCSS or SCSS...or BOTH! :D

File: index.html [hunk 3]
```diff
@@ -15,37 +16,54 @@
...
   <div class="wrapper">
     <header>
-      <h1>Focus App</h1>
+      <h1><em>focus</em></h1>
     </header>
 
     <div class="sounds__wrapper">
       <div class="sounds">
-        <div class="sounds__item">
+        <div class="sounds__item" data-description="Rain">
```
Comment (tabuckner) [inline index.html:24]: nice use of ARIA attributes

File: js/sound-player.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const playerButton = document.querySelector('.player__button');
```
Comment (tabuckner) [inline js/sound-player.js:1]: OH SHIT. NOOOOOWWWWW THE FUN STARTS 🚀

File: js/sound-player.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+const soundItems = document.querySelectorAll('.sounds__item');
+
+// Function that enables/disables a specific sound when it has been clicked
+function handleSoundItemsClick(event) {
+  const soundItem = event.currentTarget;
+  const slider = soundItem.querySelector('.slidecontainer');
```
Comment (tabuckner) [inline js/sound-player.js:7]: nit-picky comment; we might consider changing that class name to something like:
```suggestion
  const slider = soundItem.querySelector('.slide-container');
```

or

```suggestion
  const slider = soundItem.querySelector('.slide__container');
```

File: css/styles.css [hunk 6]
```diff
@@ -48,18 +50,72 @@ header {
...
+.sounds__item-active {
+  opacity: 1;
+}
+
+.slidecontainer {
+  padding-top: 15px;
+  visibility: hidden;
+}
+
+.slidecontainer-enabled {
```
Comment (tabuckner) [inline css/styles.css:72]: To more closely align with BEM modifier spec:
```suggestion
.slidecontainer--enabled {
```

File: package.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (tabuckner) [inline package.json:1]: ❗ ❗ ❗ ❗ ❗ ❗

File: package.json [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ocontrerastx/focus-app/issues"
+  },
+  "homepage": "https://github.com/ocontrerastx/focus-app#readme",
+  "devDependencies": {
+    "babel-eslint": "^9.0.0",
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb": "^17.1.1",
```
Comment (tabuckner) [inline package.json:22]: does wes-bos's config require airbnb as a peer?

File: package.json [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+    "eslint-config-prettier": "^4.3.0",
+    "eslint-config-wesbos": "0.0.19",
+    "eslint-plugin-html": "^5.0.5",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
```
Comment (tabuckner) [inline package.json:27]: that's cool

File: package.json [hunk 10]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "focus-app",
+  "version": "1.0.0",
+  "description": "Web app that allows you to play various sounds to help users focus ",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (tabuckner) [inline package.json:7]: i see that we've adding some new dev dependencies but haven't added any NPM scripts yet. 

If you're running stuff like babel as of now, what commands/etc are you using to get it done? 

Maybe we can move those into NPM scripts for ease of use and for other devs to onboard more quickly :)

File: package.json (hunk 7 above)
Comment (ocontrerastx) [reply package.json:1]: Are those bad red question marks or good ones? @tabuckner

Comment (tabuckner) [reply package.json:1]: good ones :)

**Pull request merged**